_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<a href="https://www.gnu.org/licenses/lgpl-3.0.en.html">
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+</a>
+
+This blog post outlines MDAnalysis' proposal to change its license
+to the [Lesser GNU Public License][LGPL] (LGPL v3+).
+
+A summary of our [reasons for proposing this license
+change](#Rationale-for-changing-licenses), alongside upcoming actions
+for [community members and library
+contributors](#How-will-the-relicensing-process-work?) are provided.
+
+
+> ⚠️ **Disclaimer**
+> The MDAnalysis core team members are not
+> lawyers, as such the information provided here __does not, and is not
+> intended to, constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing, it
+> purely aims to inform MDAnalysis developers and users on why
+> we believe the library should be relicensed.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Insitute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+
+## Overview
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License v2 (or any later versions)][GPLv2] (GPL v2+) to the less
```

Comment:
Nitpick but  we use GPLv2 in the intro but then use `GPL` interchangeably with `GPLv2` without making it clear that they are the same thing. 

Should we be slightly more explicit?

Reply:
I suspect it doesn't, but I've updated the text to be clearer where possible.